docs/platform-services/automation-service/app-central/integrations/sumo-logic-automation-tools.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Sumo Logic Automation Tools
+description: ''
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img src={useBaseUrl('/img/platform-services/automation-service/app-central/logos/sumo-logic-notifications.png')} alt="sumo-logic-notifications" width="100"/>
+
+***Version: 1.0  
+Updated: Sep 9, 2024***
+
+Sumo Logic Automation Tools simplifies CloudSOAR playbooks with data processing and automation.
+
+### Actions
+
+* **Buffer** (*Custom*) - [Description](#buffer)
+* **Data Transform** (*Custom*) - [Description](#data-transform)
+* **Build JSON Object** (*Custom*) - [Description](#build-json-object)
+* **Build Signal Output** (*Custom*) - [Description](#build-signal-output)
+
+## Actions Details
+
+### Buffer
+- Takes a JSON string or object and returns it as a JSON result. Helpful for dumping a JSON blob in string format and rendering in JSON format.
+
+### Data Transform
+- Provides various functions to more easily transform data in a playbook:
+  - **String Input** - The value to transform.
+  - **Transform Function** - Which string operation to use. Valid options are 'Split', 'Strip', 'Replace' or 'Regex'
+      - **Split**: Splits a string into an array based on the delimiter provided in the "Transform Argument" parameter.
+      - **Strip / lstrip / rstrip**: Removes leading and trailing characters from a string. Space is the default character to remove, unless specified otherwise in the "Transform Argument" parameter.
+      - **Append**: Attaches a string to the end of a string (String Input > Transform Argument)
+      - **Prepend**: Attaches a string to the beginning of a string (Transform Argument > String Input)
+      - **Replace**: Replace a specified string/word/character with the value specified in the "Transform Argument" and "Replace" parameters.
+      - **Regex Replace**: Replace a specified string/word/character with the value specified as a regular expression. This allows for user-provided value to be replace within the 'Transform Argument' and an user-provided value to be replace with in the 'Replace' argument.
+      - **Regex**: Match a string with a regex pattern (python regex). Currently, this will return only the first capture group/match in an array. Could be further modified to accept an array num input.
```

Comment:
```suggestion
      - **Regex**: Match a string with a regex pattern (python regex). This will return only the first capture group/match in an array.
```